### ISSUE TITLE
env_laser with dissolvetype != -1 crashes server on zombie touch 

### ISSUE THREAD

**Issue by dysphie** [nmrih/source-game#1151]
An example of this exists in `nmo_hellgate_r2`:

```cpp
entity
{
	"id" "174481"
	"origin" "-132 1055 35"
	"width" "5"
	"TextureScroll" "25"
	"texture" "sprites/laserbeam.spr"
	"targetname" "LAZ0R"
	"spawnflags" "48"
	"rendercolor" "234 130 36"
	"renderamt" "200"
	"NoiseAmplitude" "20"
	"LaserTarget" "LAZ0R_target"
	"EndSprite" "sprites/physcannon_bluecore2b.vmt"
	"dissolvetype" "1"
	"damage" "9001"
	"classname" "env_laser"
}
```
[crash_tjqvp6qrqros.zip](https://github.com/nmrih/source-game/files/4884567/crash_tjqvp6qrqros.zip)

**Comment by felis-catus** [comment]
Unable to repro with identical entdata. Crash points to closed vphysics.dll module after ragdoll creation, underlying problem might be something else.

**Comment by felis-catus** [comment]
Still unable to repro this, might be an engine issue. Closing.

**Comment by felis-catus** [comment]
Repro'd! If a zombie model is missing collision data, the crash is 100% reproducible (e.g. hunter_infected.mdl).

**Comment by felis-catus** [comment]
Resolved by #1253 closure.
